src/js/handleGame.js

```diff
@@ -0,0 +1,164 @@
+import createGameBoards from "./boardView.js";
+import {UltimateBoard} from "./Boards/UltimateBoard.js";
+import {HumenPlayer} from "./Players/HumenPlayer.js";
+import {ComputerPlayer} from "./Players/ComputerPlayer.js";
+import { Board } from "./Boards/Board.js";
+
+const   boardSize = 3,
+        elementsID = {gameStatus: 'gameStatus',
+                    humPlayersign: 'humPlayerSign',
+                    compPlayersign: 'compPlayerSign'};
+let humPlayer,
+    compPlayer,
+    currPlayer,
+    bigBoard,
+    scoreBrd;
+
+
+export function startGame(mainBoardElem, scoreElem, gameInfoElem){
+    initGameBoards(mainBoardElem, scoreElem);
+    initPlayers();
+    initGameInfo(gameInfoElem);
+    updateCurrPlayer();
+}
+
+function initGameBoards(mainBoardElem, scoreElem){
+    // Creating all game boards
+    const brds = createGameBoards(boardSize);
+
+    mainBoardElem.appendChild(brds.mainBrd.tblElem);
+    scoreElem.appendChild(brds.scoreBrd.tblElem);
+
+    bigBoard = new UltimateBoard("brd-0", boardSize, brds.mainBrd.tblElem, brds.mainBrd.cellsArr);
+    scoreBrd = new Board("scrBrd", boardSize, brds.scoreBrd.tblElem, brds.scoreBrd.cellsArr);
+
+    addClickEventCells();
```

Comment:
This file makes it seem like you didn't separate "Game" class from "GameView" class.
What would happen when you start writing the NodeJS code? You won't be able to reuse the "Game" class as is, separating from any UI presentation logic